BaseFilter/sections/foreign.txt

```diff
@@ -3092,6 +3092,9 @@ bb.is$$div[class="banner"]
 !---------------------------------------------------------------------
 ! NOTE: Indian
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176683
+irctc.co.in###cube
```

Comment:
I think that script which loads this floating cube also could be blocked:
```adblock
||cube.nlpcaptcha.in/index.php/cubes/getCubeBox/$domain=irctc.co.in
```
So `iframe` wouldn't be loaded, but cosmetic rule `irctc.co.in###cube` is still needed, otherwise there will be invisible leftover which will block part of the website.